pom.xml

```diff
@@ -279,8 +298,55 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>net.revelc.code.formatter</groupId>
+          <artifactId>formatter-maven-plugin</artifactId>
+          <version>2.11.0</version>
+          <configuration>
+            <configFile>${format.directory}/eclipse-format.xml</configFile>
+            <skip>${formatter.skip}</skip>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>${formatter.goal}</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>net.revelc.code</groupId>
+          <artifactId>impsort-maven-plugin</artifactId>
+          <version>1.3.2</version>
+          <configuration>
+            <groups>java.,javax.,org.,com.</groups>
```

Comment:
@rsynek This duplicates the content of `eclipse.importorder` - why?

Reply:
There is no way of passing the file into this plugin and at the same time, the formatter plugin does not care about imports. So yes, there is a duplication of the imports between the maven plugin use case and the IDE use case.